src/postselection.jl

```diff
@@ -141,98 +127,116 @@ function initByMsmt_2D(data,Anum,numSegments,numMeas_A,indMeas,selectMeas,select
 
   ind0=indMeas[1,:]
   ind1=indMeas[2,:]
+  nbins = 500
+
+  if ~isa(Anum, Array)
+    Anum = [Anum]
+  end
+  if ~isa(threshold, Array)
+    threshold = repmat([threshold],length(Anum), 1)
+  end
```

Comment:
As a general style thing, I'd suggest handling this kind of type manipulation through multiple method definitions. Make the primary definition of `initByMsmt_2D` take in a `Anum::Vector{Int}` and a `threshold::Vector{Float64}`, then make another version which takes in `Anum::Int` and `threshold::Float64` and simply calls `initByMsmt_2D` with those variables wrapped in appropriate arrays.

Another thing to point out here, is that Julia has a `repeat` method which can replace many uses of `repmat`, like this one.